src/boot/z_std_dma.c

```diff
@@ -380,6 +376,15 @@ void DmaMgr_ProcessRequest(DmaRequest* req) {
         if (vrom >= iter->file.vromStart && vrom < iter->file.vromEnd) {
             // Found the region this request falls into
 
+#if PLATFORM_N64
+            // Based on the MM Debug ROM, these strings are part of the condition for the empty if statement below,
+            // as `... && DmaMgr_StrCmp("", "kanji") != 0 && DmaMgr_StrCmp("", "link_animetion") != 0`
+            (void)"";
+            (void)"kanji";
+            (void)"";
+            (void)"link_animetion";
+#endif
+
             if (0) {
```

Comment:
if we actually wrote the conditions as 0 && ... would it properly optimize out the calls?

Reply:
I think so, but it's a bit tricky to make this look good since these strings are N64-only. Maybe it could be
```c
#if PLATFORM_N64
if (0 && && DmaMgr_StrCmp("", "kanji") != 0 && DmaMgr_StrCmp("", "link_animetion") != 0)
#else
if (0)
#endif
{
    PRINTF("DMA ROM:%08X RAM:%08X SIZE:%08X %s\n", vrom, ram, size, filename);
}
```
or
```c
#if PLATFORM_N64
if (0 && && DmaMgr_StrCmp("", "kanji") != 0 && DmaMgr_StrCmp("", "link_animetion") != 0) {
    PRINTF("DMA ROM:%08X RAM:%08X SIZE:%08X %s\n", vrom, ram, size, filename);
}
#else
if (0) {
    PRINTF("DMA ROM:%08X RAM:%08X SIZE:%08X %s\n", vrom, ram, size, filename);
}
#endif
```
What do you suggest?

Reply:
I was moreso just curious if it would work, not entirely sure what would be best or if we should even do this at all. I think its neat to document how the strings would have been used, but the comment does that just as well I suppose